tests/python/gpu/test_operator_gpu.py

```diff
@@ -2507,6 +2507,18 @@ def test_arange_like_dtype():
         for v in out:
             assert v.dtype == t
 
+# do not remove seed
+@with_seed(1234)
+def test_dropout_with_seed():
+    a = mx.nd.ones((10, 10), ctx=mx.gpu())
+    dropout = mx.gluon.nn.Dropout(0.5)
+    with mx.autograd.record():
```

Comment:
I have a suggestion.
Firstly, replace `@with_seed(1234)` with `@with_seed()`.
Then add the following code,
```python
info = np.iinfo(np.int64)
seed = np.random.randint(info.min, info.max)
```
Finally, we add `mx.random.seed(seed)` before each `dropout(a)`.

Reply:
We can move the unittest into `tests/python/unittest/test_operator.py`, since `test_dropout_with_seed` is also necessary for CPU.